src/TypeShape.Roslyn/version.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
+  "inherit": true,
+  "assemblyVersion": {
```

Comment:
Presumably this is refining the assembly version for `TypeShape.Roslyn` specifically? What is the default assembly version for the others?

Reply:
The default is to take major.minor from the `version` property for the assembly version.